src/browser/provider/built-in/dedicated/base.js

```diff
@@ -29,6 +35,33 @@ export default {
         return this.openedBrowsers[browserId].config.headless;
     },
 
+    _getCropDimensions (viewportWidth, viewportHeight) {
+        if (!viewportWidth || !viewportHeight)
+            return null;
+
+        return {
+            left:   0,
+            top:    0,
+            right:  viewportWidth,
+            bottom: viewportHeight
+        };
+    },
+
+    async takeScreenshot (browserId, path, viewportWidth, viewportHeight) {
+        const runtimeInfo   = this.openedBrowsers[browserId];
+        const browserClient = this._getBrowserProtocolClient(runtimeInfo);
+        const binaryImage   = await browserClient.getScreenshotData(runtimeInfo);
+        const pngImage      = await readPng(binaryImage);
+
+        const croppedImage = await cropScreenshot(pngImage, {
+            path,
+
+            cropDimensions: this._getCropDimensions(viewportWidth, viewportHeight)
```

Comment:
Just a formatting moment.
I would like to create a constant `cropDimensions` and pass arg as 
```JS
{
    path,
    cropDimensions
}
```
This empty line confuses me.
Of course it's not required